bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -276,12 +276,23 @@ class BinaryContext {
   void deregisterSectionName(const BinarySection &Section);
 
 public:
+  enum class ICFLevel {
+    None,
+    Safe,
```

Comment:
I think the comment here was useful, we can keep it (without mentioning MSVC). Thanks.